skyportal/models.py

```diff
@@ -66,9 +66,12 @@ def process_result_value(self, value, dialect):
 class Group(Base):
     name = sa.Column(sa.String, unique=True, nullable=False)
 
-    streams = relationship('Stream', secondary='stream_groups',
+    streams = relationship('Stream', secondary='group_streams',
```

Comment:
Changed the order since I think in terms of adding stream access to groups rather than adding groups to streams.